sailfishusermanager.cpp

```diff
@@ -34,11 +35,20 @@ const int HOME_MODE = 0700;
 const int QUIT_TIMEOUT = 60 * 1000; // One minute quit timeout
 const int MAX_RESERVED_UID = 99999;
 const int OWNER_USER_UID = 100000;
+const auto SYSTEMD_MANAGER_DEST = QStringLiteral("org.freedesktop.systemd1");
+const auto SYSTEMD_MANAGER_PATH = QStringLiteral("/org/freedesktop/systemd1");
+const auto SYSTEMD_MANAGER_INTERFACE = QStringLiteral("org.freedesktop.systemd1.Manager");
+const auto SYSTEMD_MANAGER_START = QStringLiteral("StartUnit");
+const auto SYSTEMD_MANAGER_STOP = QStringLiteral("StopUnit");
+const auto USER_SERVICE = QStringLiteral("user@%1.service");
+const auto SYSTEMD_MANAGER_REPLACE = QStringLiteral("replace");
+const auto AUTOLOGIN_SERVICE = QStringLiteral("autologin@%1.service");
 
 static_assert(SAILFISH_UNDEFINED_UID > MAX_RESERVED_UID,
               "SAILFISH_UNDEFINED_UID must be in the valid range of UIDs");
 
-SailfishUserManager::SailfishUserManager(QObject *parent) : QObject(parent), m_lu(new LibUserHelper())
+SailfishUserManager::SailfishUserManager(QObject *parent) : QObject(parent), m_lu(new LibUserHelper()), m_switchUser(0),
```

Comment:
I think you should split this to multiple lines like we have in most projects. Also this doesn't seem to initialize m_currentUid and I don't see LibUserHelper being destructed.

Reply:
Ok